fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/api/resource/conditions/v1/DefaultResourceConditions.java

```diff
@@ -216,6 +216,24 @@ public static <T> ConditionJsonProvider registryContains(RegistryKey<Registry<T>
 		return ResourceConditionsImpl.registryContains(REGISTRY_CONTAINS, registry.getValue(), entries);
 	}
 
+	/**
+	 * Creates a condition with a directory field, for use in overlay conditions.
+	 */
+	public static ConditionJsonProvider withDirectory(ConditionJsonProvider condition, String directory) {
```

Comment:
Where is this used/tested? It might be nice to show how to datagen these if possibe? It might not be a bad idea to rename this as its only used for overlays?

Reply:
Opting to remove this as there doesn't appear to be a way to datagen `pack.mcmeta` files.